editors/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
```

Comment:
extends the main tsconfig to include dom libs in the editors. dom libs are not included in the document models. I checked that the behavior is indeed different in each directory